src/store/reducers/index.js

```diff
@@ -6,7 +6,7 @@ import status from './connection'
 import Hoodie from '@hoodie/client'
 const hoodieInstance = new Hoodie({
   url: window.location.origin,
-  PouchDB: require('pouchdb-browser')
+  PouchDB: require('pouchdb-browser').default
```

Comment:
I did not think so either, but the change from 6.2.0 -> 6.3.x required it to work. I did not isolate the exact patch responsible